frontend_lib/src/mentionOrLink.js

```diff
@@ -23,44 +22,7 @@ export const LINK_REGEX = /#([0-9]+)(?=\s|$)/
 export const LINK_TAG_NAME = 'a'
 export const LINK_CLASS = 'internal_link primaryColorFont'
 
-export const GROUP_MENTION_TRANSLATION_LIST = ['all', 'tous', 'todos', 'alle', 'الكل']
-
-const wrapMentionsFromText = (text, doc, invalidMentionList) => {
-  // INFO - GB - 2022-02-28 - takes a text as string, and returns a document fragment
-  // containing this text, with tags added for the mentions
-  // The second RegEx support arabic group mention
-  const matchMention = text.match(MENTION_REGEX)
-  const matchArabic = text.match(/@(الكل)(?=\s|$)/)
-  let match
-
-  if (matchArabic && matchMention) {
-    match = matchArabic.index < matchMention.index ? matchArabic : matchMention
-  } else match = matchArabic || matchMention
-
-  if (!match || (match.index > 0 && (text[match.index - 1].trim()))) {
-    return doc.createTextNode(text)
-  }
-
-  const fragment = doc.createDocumentFragment()
-
-  fragment.appendChild(doc.createTextNode(text.substring(0, match.index)))
-
-  if (invalidMentionList.indexOf(match[0]) === -1) {
-    const wrappedMention = doc.createElement(MENTION_TAG_NAME)
-    wrappedMention.className = MENTION_CLASS
-    wrappedMention.id = `${MENTION_ID_PREFIX}${uuidv4()}`
-    wrappedMention.textContent = match[0]
-    fragment.appendChild(wrappedMention)
-  } else {
-    const notWrappedMention = doc.createTextNode(match[0])
-    fragment.appendChild(notWrappedMention)
-  }
-
-  const mentionEndIndex = match.index + match[0].length
-  fragment.appendChild(wrapMentionsFromText(text.substring(mentionEndIndex), doc, invalidMentionList))
-
-  return fragment
-}
+const GROUP_MENTION_TRANSLATION_LIST = ['all', 'tous', 'todos', 'alle', 'الكل']
```

Comment:
I think this behavior should be documented somewhere, to explain that if we add a new language we must add something here.
Probably in a translation related documentation (like frontend/doc/i18n.md) or anywhere you see fit

Reply:
This is documented somewhere (can't find it tho)
However, I didn't add it, and this is destinated to be removed - as you can see, I removed the ``export``. Since it's not used. I didn't fully remove it because it's called in HTML-Note, and as a quick fix I didn't want to break everything. I can spend more time to fully remove it.

Reply:
indeed, it's ok then, you can resolve